android/java/org/chromium/chrome/browser/sync/settings/BraveManageSyncSettings.java

```diff
@@ -14,16 +14,16 @@
 
 import androidx.annotation.Nullable;
 import androidx.annotation.VisibleForTesting;
-import androidx.fragment.app.FragmentManager;
 import androidx.preference.Preference;
 
 import org.chromium.base.CommandLine;
 import org.chromium.base.ContextUtils;
 import org.chromium.base.Log;
 import org.chromium.chrome.R;
-import org.chromium.chrome.browser.password_manager.settings.ReauthenticationManager;
+// TODO(alexeybarabash): backport PasswordAccessReauthenticationHelper
```

Comment:
Fixed at  [[CR140][Android] Backport built-in Password Manager #30433](https://github.com/brave/brave-core/pull/30433)